retention/src/test/java/org/apache/falcon/retention/FeedEvictorTest.java

```diff
@@ -462,6 +462,43 @@ public void testFeedBasePathExists() throws Exception {
         }
     }
 
+    @Test
+    public void testEvictionStatsMetaWithNoPattern() throws Exception {
+        try {
+            Configuration conf = cluster.getConf();
+            FileSystem fs = FileSystem.get(conf);
+            fs.delete(new Path("/"), true);
```

Comment:
Can you please make that change now? It should not take long and it would save us from potential pain in the future.